Paper.tex

```diff
@@ -755,7 +755,7 @@ \section{Message Call} \label{ch:call}
 0 & \text{if} \quad \boldsymbol{\sigma}^{**} = \varnothing \\
 g^{**} & \text{otherwise}
 \end{cases} \\
-(\boldsymbol{\sigma}^{**}, g^{**}, \mathbf{s}, \mathbf{o}) & \equiv & \begin{cases}
+\qquad (\boldsymbol{\sigma}^{**}, g^{**}, A, \mathbf{o}) & \equiv & \begin{cases}
```

Comment:
The equation numbering overlaps with the equation. This just scoots things over a bit. If theres a more elegant solution than qquad i'd be happy to do that instead.